### ISSUE TITLE
Server Lag

### ISSUE THREAD

**Issue by skittishtrigger** [skittishtrigger/TechnAintEasy#2]
After updating to kcauldron(and forge in modpacks) v1448 severe tps lag is induced into server. It points to worldgen conflictions within jas.

Original post:

I recently updated my forge to 1448 (this is the only thing that has changed recently) before this there was 0 tps lag. My average tps over a 48h period was 19.48 with spikes down as low as 17. I ran a /sampler start/stop/export (Don't remember how long but it was at min an hour cause I forgot I had it running) and tracked the lag down to http://i.imgur.com/yhc9gQR.jpg . To download the full nps file I hosted it for you at http://solder.storagedragon.info/modpack/mods/lagcheck.nps . Hopefully it is something I have overlooked or missed. would appreciate your advice or any info you can provide so I can squelch this.

tl:dr

issue: http://i.imgur.com/yhc9gQR.jpg
full log: http://solder.storagedragon.info/modpack/mods/lagcheck.nps
mods list: http://www.technicpack.net/modpack/technainteasy.650138/mods
Forge: 10.13.4.1448
FML: v7.10.99.99
MCP: v9.05
Server: Ubuntu 14.10 headless (dedicated)
Ram: 10g

I hope that is enough/correct info.
As an addendum, I should probably check what forge/java version this is built against.

**Comment by bochen415** [comment]
https://github.com/ProjectZulu/JustAnotherSpawner/issues/140

**Comment by skittishtrigger** [comment]
Upon further evaluation this might/seems to be caused by the forge update itself.  Research into mods requiring the version 1448 is needed and a downgrade in forge version to be tested.

**Comment by skittishtrigger** [comment]
This seems to only happen (the tps drops exponentially) while exploring project zulu dungeons and battle towers dungeons that generate inverted (underground) instead of above ground.

**Comment by skittishtrigger** [comment]
This has been confirmed by testing tps over extended periods. Lag only happens while exploring with these mods (zulu/jas) generating.

**Comment by skittishtrigger** [comment]
Since there has been no response yet from my other avenues of investigation I pushed a server and modpack update to forge 1481. Further information will be coming as I test and see what else is going on.

**Comment by skittishtrigger** [comment]
JAS was kind enough to push an update for this. with config changes and the update it seems good so far. Closing this for now pending more data. will add tag fixed when more data supports the fix.
